src/components/Message.tsx

```diff
@@ -61,7 +64,9 @@ const Message: FC<Props> = ({ message, currentMember, member }) => {
         {message.body}
       </Typography>
       <Typography variant="caption" className={classes.time}>
-        {time}
+        {`${
+          message.updatedAt ? t(CHATBOX.MESSAGE_MODIFIED_INDICATOR) : ''
+        } ${time}`}
```

Comment:
You could use options in i18n if you think it's better https://www.i18next.com/translation-function/nesting